src/playlist/playlist.cpp

```diff
@@ -1388,7 +1388,7 @@ QString Playlist::column_name(Column column) {
     case Column_Year:
       return tr("Year");
     case Column_OriginalYear:
-      return tr("Original year");
+      return tr("Year - original"); // Done like this so sorts beside "Year" when editing smart playlists
```

Comment:
Can you make this a translator comment so translators know to try to do this for their language too? https://doc.qt.io/qt-5/i18n-source-translation.html#translator-comments

Reply:
Right, done